gdal/doc/source/programs/gdaltindex.rst

```diff
@@ -105,6 +105,12 @@ into the same output projection:
 
     gdaltindex -t_srs EPSG:4326 -src_srs_name src_srs tile_index_mixed_srs.shp *.tif
 
+- Make a tile index from files listed in a text file :
```

Comment:
For formatting consistency, the 2 above paragraphs should also have a leading '- ' on their first line and have two leading extra spaces on the next lines (before the example itself)

Reply:
Sure, updated accordingly.